src/commands/init.ts

```diff
@@ -71,6 +72,15 @@ const choices = [
     checked: false,
   },
 ];
+
+if (isEnabled("internaltesting")) {
```

Comment:
please create a new experiment (e.g. `frameworks`). `internaltesting` is for another experiment.

Reply:
done